.gitlab-ci.yml

```diff
@@ -261,14 +263,12 @@ ci-elpi:
 ci-equations:
   <<: *ci-template
 
-ci-geocoq:
+ci-fiat-crypto:
   <<: *ci-template
-  allow_failure: true
-
-# ci-fiat-crypto:
-#   <<: *ci-template
-#   # out of memory error
-#   allow_failure: true
+  tags:
+    - fast-runner
+  # out of memory error
+  # allow_failure: true
```

Comment:
You don't need to keep allow_failure comments.
Do comment the fast-runner tag on fiat-crypto with `runs out of memory on shared runners` or some such.